actors/professionPreviewer.js

```diff
@@ -1,8 +1,12 @@
 import { DataProvider } from "./dataProvider.js";
 
 export class ProfessionPreviewer {
-    constructor(dataProvider) {
-        this.dataProvider = dataProvider;
+    dataProvider = null;
+    profession = null;
```

Comment:
members require a newline between them. (I don't have an automatic action for ESLint, but something to remember)

Reply:
Fixed